site/_resources/release-process.md

```diff
@@ -103,14 +103,14 @@ Authenticating with existing credentials...
 Login Succeeded
 # The make target will do what you need
 # If you don't set the version, it will be a noop
-$ env LATEST_VERSION=v1.0.0 make tag-latest
-docker pull projectcontour/contour:v1.0.0
-v1.0.0: Pulling from projectcontour/contour
+$ env REGISTRY=docker.io/projectcontour LATEST_VERSION=v1.0.0 make tag-latest
```

Comment:
No need to change this is you don't want to, but this would be marginally better since env vars are a bit fragile with make.

```
$make tag-latest REGISTRY=docker.io/projectcontour LATEST_VERSION=v1.0.0
```

Reply:
That's a good idea, thanks.